src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -2,13 +2,17 @@
 
 ## Unreleased
 
+## 0.3.0-alpha.1
```

Comment:
```suggestion
## 1.0.0-beta.1
```

As this looks way better than an alpha!

Reply:
I'll have to go and remove the options later. Not sure if having breaking change between 1.0.0-beta.1 and 1.0.0 a good idea?

Reply:
@reyang maybe its time to add the "major refactoring" changelog entry before this release.

Reply:
unless it is 1.0.0, its okay to have breaking change. Alpha was right usage when we started. Now that this was received reviews from .NET runtime owners, GC owners etc, its good to make it beta. This will encourage more people to try it out in PROD, and give us more feedback.

Reply:
Yeah!

Reply:
Sounds good. I've updated it.